en/docs/deploy-with-wso2-choreo.md

```diff
@@ -0,0 +1,7 @@
+# Deploy with WSO2 Choreo
+
+You can deploy the Ballerina projects you create in [WSO2 Choreo](https://wso2.com/choreo/). The combination of the Ballerina VS Code extension and [Choreo VS Code extension](https://marketplace.visualstudio.com/items?itemName=WSO2.choreo) enables this functionality. 
+
+For detailed information, see the [WSO2 Choreo documentation](https://wso2.com/choreo/docs/develop-components/develop-components-using-vs-code/).
+
+![Choreo Deployment](./img/choreo-integration/choreo-integration.png){cInlineImage-half}
```

Comment:
This image contains improper service names, use a real world samples. Update the same image in home page too.

Reply:
Updated.